src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -225,6 +225,11 @@ public string PathPrefix
         /// </summary>
         public string InnerErrorComplexTypeName { get; set; } = "InnerError";
 
+        /// <summary>
+        /// Gets/Sets a value indicating whether or not to use restrictions annotations to generate paths for complex properties.
+        /// </summary>
+        public bool UseRestrictionAnnotationsToGeneratePathsForComplexProperties { get; set; } = true;
```

Comment:
UseRestrictionAnnotationsToGenerateComplexPropertyPaths

Reply:
by default, it should be "false" right?

Reply:
I agree this is a better name: `UseRestrictionAnnotationsToGenerateComplexPropertyPaths`

Reply:
> by default, it should be "false" right?

Please refer to the discussion in the issue: https://github.com/microsoft/OpenAPI.NET.OData/issues/176